src/backend/marsha/bbb/urls.py

```diff
@@ -4,16 +4,26 @@
 
 from rest_framework.routers import DefaultRouter
 
-from .api import MeetingViewSet
-from .views import MeetingLTIView
+from .api import ClassroomViewSet
+from .views import ClassroomLTIView
 
 
 app_name = "bbb"
 
 router = DefaultRouter()
-router.register("meetings", MeetingViewSet, basename="meetings")
+router.register("classrooms", ClassroomViewSet, basename="classrooms")
+router.register("meetings", ClassroomViewSet, basename="meetings")
```

Comment:
:+1:  can you add a comment to remember to remove this route later ?

Reply:
Unfortunately I'm not sure we should remove them, as maybe an LMS points to /lti/meetings/… and will keep it.

Reply:
luckily, the interest of a BBB meeting is limited in time in its current form.... So we will be able to remove it in a few days!

Reply:
Good point!